staubli_tx2_60_gazebo/package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>staubli_tx2_60_gazebo</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial Gazebo support package for the Staubli TX260 (and variants).
+    </p>
+    <p>
+      This package contains the configuration data and launch files required
+      to simulate the Staubli TX60 and the Staubli TX60L manipulator in Gazebo.
+    </p>
+    <p>
+      Before using any of the configuration files included in this package, be
+      sure to check they are correct for the particular robot model and
+      configuration you intend to use them with.
+    </p>
+  </description>
+  <author>Fetullah Atas</author>
+  <maintainer email="g.a.vanderhoorn@tudelft.nl">G.A. vd. Hoorn (TU Delft Robotics Institute)</maintainer>
+  <license>Apache2.0</license>
+
+  <url type="website">http://wiki.ros.org/staubli_experimental</url>
+  <url type="bugtracker">https://github.com/ros-industrial/staubli_experimental/issues</url>
+  <url type="repository">https://github.com/ros-industrial/staubli_experimental</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <exec_depend>gazebo_ros</exec_depend>
+  <exec_depend>gazebo_ros_control</exec_depend>
+  <exec_depend>gazebo_ros_pkgs</exec_depend>
+  <exec_depend>joint_state_controller</exec_depend>
+  <exec_depend>joint_trajectory_controller</exec_depend>
+  <exec_depend>position_controllers</exec_depend>
+
```

Comment:
This now seems to be missing an `exec_depend` on the TX2-60 support package.

Only the `build_depend` should have been removed, as it's not neded.